src/styles/block/nav.scss

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  align-items: center;
+  height: $header-height;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should use single quotes in SCSS files instead of double quotes. Please replace "Roboto, sans-serif" with 'Roboto, sans-serif'.